### ISSUE TITLE
need any help ?

### ISSUE THREAD

**Issue by kevinpaul468** [saivishnu725/OnlyThighs#1]

**Comment by saivishnu725** [comment]
Yes. I had stopped working on this due to exams. But I do need help with the UI development (I suck at web dev after a point)

**Comment by kevinpaul468** [comment]
im no better i came for this looking for projects to apply nodejs
//just learnt node

**Comment by saivishnu725** [comment]
Try to upload images/videos/files to AWS S3 using their node package. Also, express, ejs are fun to learn. I'd recommend using Nodemon for the server instead of running it directly on the file. I have saved so much time when I learned that this exists.

As for the help, I would appreciate some help. I keep going off the track now and then. I need help mostly with the UI part. I forgot all about Flexbox and it is hurting me now. I might watch some tutorials soon. I am currently busy with the [portfolio-v2](https://github.com/saivishnu725/portfolio-v2023.12).

**Comment by kevinpaul468** [comment]
i dont even know wtf a nodeman is

**Comment by saivishnu725** [comment]
>  nodeman
> 

[Nodemon](https://www.npmjs.com/package/nodemon) is a tool that helps develop Node. js based applications by automatically restarting the node application when file changes in the directory are detected


Life changer fr

**Comment by kevinpaul468** [comment]
Ahh ,i get it now 

It all makes sense to me now
